.github/workflows/chromium-validation.yaml

```diff
@@ -0,0 +1,33 @@
+name: Validate Latest Chromium Binary PDF generation
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-unit-tests:
+    name: Run Unit Tests with Latest Chrome
```

Comment:
We are not running unit tests, but an integration test with Chromium. A unit test only considers the behavior of a piece of code "by-itself", meaning without any dependencies. That's why in the unit test for the PI Invoice, we [mock out the Chromium binary](https://github.com/CCI-MOC/invoicing/blob/7d78602b7e7a99adee4f848211ad64537b111e35/process_report/tests/unit/invoices/test_pi_specific_invoice.py#L110).